### ISSUE TITLE
Context Menu upload set as default action for images in Windows 10

### ISSUE THREAD

**Issue by snoozbuster** [bkeiren/EasyImgur#35]
Hey, guys! Love the work you've done with the project since I last saw it, especially cleaning up those naming conventions (I didn't want to say anything... ;P). I ran into something weird recently, I noticed that the context menu options had been set as the default options for my images, as per the image:

![Context menu weirdness](http://i.imgur.com/uT7hikK.png)

I'm not sure if this is because I broke the configuration on my system specifically, or if it's because the registry tweaks I was using to get cheap context-menu support have changed on Windows 10. If I recall correctly, the registry entries I'm using I found primarily through trial-and-error, I don't remember if there's really much official documentation on the topic and I certainly don't think it's the preferred way to do context-menu tweaks. Unfortunately, the preferred way is to write shell extensions in C++ and install the resulting DLL (although, you get way more features, such as being able to run actual processing on things that are right-clicked to determine what they are and how to handle them, flyout menus, and more), and that was just too much to be necessary for a few quick features.

Anyway, long story short the context menus in general might be something worth revisiting. My finals are coming up pretty soon and then I'll be graduating so I'm a bit strapped for time right now, but I'll try to eventually get around to this if no one else does.
